autoload/go/config.vim

```diff
@@ -273,11 +273,19 @@ function! go#config#MetalinterCommand() abort
   return get(g:, "go_metalinter_command", "golangci-lint")
 endfunction
 
+function! go#config#MetalinterUseLocalConfig() abort
+  return get(g:, "go_metalinter_use_local_config", 0)
+endfunction
+
 function! go#config#MetalinterAutosaveEnabled() abort
   let l:default_enabled = ["vet", "golint"]
 
   if go#config#MetalinterCommand() == "golangci-lint"
-    let l:default_enabled = ["govet", "golint"]
+    if go#config#MetalinterUseLocalConfig() == 0
+      let l:default_enabled = ["govet", "golint"]
+    else
+      let l:default_enabled = []
+    endif
```

Comment:
Modify this and other similar changes to be more in line with usual go standards that you may already be familiar with:
```
let l:default_enabled = ["govet", "golint"]
if go#config#MetalinterUserLocalConfig()
  let l:default_enabled = []
endif
```